.aegir.js

```diff
@@ -1,5 +1,13 @@
 export default {
   tsRepo: false,
+  dependencyCheck: {
+    input: [
+      'dist/**/*.js',
+    ],
+    productionInput: [
+      'dist/**/*.js',
+    ],
+  },
```

Comment:
Created an issue for this: https://github.com/ipfs/aegir/issues/1104

Reply:
Left a comment on https://github.com/ipfs/aegir/issues/1104#issuecomment-1301956533